controllers/drinks.js

```diff
@@ -320,13 +319,58 @@ const getFavoriteDrink = async (req, res, next) => {
 const getDrinkById = async (req, res, next) => {
   const { drinkId } = req.params;
 
-  const response = await Recipe.findById(drinkId);
+  const response = (await Recipe.findById(drinkId)).toObject();
```

Comment:
const response = await Recipe.findById(drinkId);

const resObj = response.toObject();

I think better to do it this way.